src/scripts/stringUtils.ts

```diff
@@ -1,22 +1,35 @@
 import {ObjectUtils} from "./objectUtils";
 import {Localization} from "./localization/localization";
 
+import {Status} from "./clipperUI/status";
+
 import * as _ from "lodash";
 
 export module StringUtils {
+	export interface ParsedPageRange {
+		status: Status;
```

Comment:
I originally had a different enum called `OperationResult` which was just `Succeeded', `Failed`. That is a fair point about `InProgress`. It should be quick enough converting it back.